### ISSUE TITLE
Build error after upgrading config-less to 1.1.2

### ISSUE THREAD

**Issue by smax48** [easy-webpack/config-less#5]
After upgrading easy-webpack/config-less to the latest version 1.1.2, I get the following build error:

```
ERROR in ./wwwroot/assets/styles/bis.less                                                                                                                                                     
Module parse failed: <...>\wwwroot\assets\styles\bis.less Unexpected character '@' (1:1)                                                                        
You may need an appropriate loader to handle this file type.                                                                                                                                  
| ﻿@bis-image-folder: "./images";                                                                                                                                                              
|                                                                                                                                                                                             
| @import "../../less/variables.less";                                                                                                                              
 @ ./app/main.ts 20:0-44                                                                                                                                                                      
 @ ./app ^\.\/.*$                                                                                                                                                                             
 @ ./~/aurelia-loader-webpack/dist/commonjs/aurelia-loader-webpack.js                                                                                                                         
 @ multi aurelia  
```

**With v1.1.1 everything works fine**

I have the following configuration for less in webpack config:
```
      require('@easy-webpack/config-less')
        ({ filename: 'bis.css', allChunks: true, sourceMap: true }),
```

In my main.ts, I have:
```
import "../wwwroot/assets/styles/bis.less";
```

**Comment by niieani** [comment]
Yes, this is the same as https://github.com/easy-webpack/config-sass/issues/11.
I'm really sorry about this, I made a patch release instead of a major one. It's because the new version of config-css is only supported by the latest version of Webpack. I'll try to republish under a major.

**Comment by smax48** [comment]
np :)
Until the latest webpack is supported by aurelia, I can lock config-less to 1.1.1

**Comment by niieani** [comment]
Re-released as breaking change v2.0.0. 

v1.1.2 unpublished.
